ext/date/php_date.stub.php

```diff
@@ -692,6 +693,26 @@ public function __wakeup(): void {}
     public static function __set_state(array $array): DateInterval {}
 }
 
+final class DateTimeStringInterval extends DateInterval
+{
+    public readonly ?string $date_string;
```

Comment:
Yes, using camelCase was better, but this is not easily possible because `DateTimeStringInterval` extends the original class, and because of BC concerns. I'm fine to rename the property if @derickr is ok with it. We could also create a new `$dateString` property besides `$date_string`, and in the same time deprecate the latter.